### PR TITLE
Update Cody Enterprise plan

### PR DESCRIPTION
**Pull request by iskyOS**
Updates the Cody subscription page for changes to the Enterprise plan.

## Test plan

@thenamankumar will run `sg start dotcom` locally and then manually QA https://sourcegraph.test:3443/cody/subscription.

### CODE REVIEW

File: client/web/src/cody/subscription/CodySubscriptionPage.tsx [hunk 1]
```diff
@@ -401,7 +401,7 @@ export const CodySubscriptionPage: React.FunctionComponent<CodySubscriptionPageP
                                 <Text weight="bold" className="mb-3">
                                     Support
                                 </Text>
-                                <Text className="d-inline text-muted">Community support through Discord</Text>
+                                <Text className="d-inline text-muted">Support with limited SLAs</Text>
```
Comment (chenkc805) [inline client/web/src/cody/subscription/CodySubscriptionPage.tsx:404]: Is this for Cody Pro? if so, maybe worth saying "Email support with limited SLAs"?

Comment (iskyOS) [reply client/web/src/cody/subscription/CodySubscriptionPage.tsx:404]: It is, yes - there is portal + email support now. I can make that update.

Comment (iskyOS) [reply client/web/src/cody/subscription/CodySubscriptionPage.tsx:404]:
```suggestion
                                <Text className="d-inline text-muted">Email support with limited SLAs</Text>
```

**Review by chenkc805 — APPROVED** [review]

**Comment by iskyOS** [comment]
@chenkc805 Do you know how to test this before it goes live? I need to add a test plan but honestly not sure how sourcegraph.com gets tested

**Comment by chenkc805** [comment]
> @chenkc805 Do you know how to test this before it goes live? I need to add a test plan but honestly not sure how sourcegraph.com gets tested

I don't unfortunately, you'll have to ask the eng team on this @thenamankumar @vdavid can you advise/help?

**Comment by thenamankumar** [comment]
@iskyOS you can run `sg start dotcom` locally and then test it on `sourcegraph.test:3443/cody/subscription`.

**Comment by vdavid** [comment]
@thenamankumar, could you execute the test plan and if it looks good, merge the PR? I don't think Alex has the dev env, and mine still doesn't work for the monorepo.

**Comment by thenamankumar** [comment]
image

**Pull request merged**